packages/terra-icon/src/Icon.scss

```diff
@@ -0,0 +1,5 @@
+/* rtl:ignore */
+@keyframes spin {
+  0% { transform: rotate(0deg); }
+  100% { transform: rotate(359deg); }
+}
```

Comment:
Is it possible to move this outside of the `:local` block and keep it within the same scss file or are there special things that happen to `.module.scss` files?